scripts/build-circt.sh

```diff
@@ -69,15 +69,16 @@ git checkout ${GIT_COMMIT}
 cmake -G Ninja \
 	${CIRCT_GIT_DIR} \
 	-B ${CIRCT_BUILD_DIR} \
-	-DCMAKE_C_COMPILER=clang-16 \
-	-DCMAKE_CXX_COMPILER=clang++-16 \
+	-DCMAKE_C_COMPILER=clang-17 \
+	-DCMAKE_CXX_COMPILER=clang++-17 \
 	-DCMAKE_BUILD_TYPE=RelWithDebInfo \
-	-DLLVM_DIR=/usr/lib/llvm-16/cmake/ \
-	-DMLIR_DIR=/usr/lib/llvm-16/lib/cmake/mlir \
+	-DLLVM_DIR=/usr/lib/llvm-17/cmake/ \
+	-DMLIR_DIR=/usr/lib/llvm-17/lib/cmake/mlir \
```

Comment:
Should those absolute paths not come from `llvm-config`` At least the path prefix?

Reply:
Using llvm-config